cmd/nerdctl/stats.go

```diff
@@ -318,6 +318,9 @@ func statsAction(cmd *cobra.Command, args []string) error {
 		}
 
 		for _, c := range ccstats {
+			if c.ID == "" {
```

Comment:
is it possible to have empty id ?

Reply:
container with empty id is useless here, I guess

Reply:
We cannot have an empty ID.
But it looks like `c.ID`  is useless in the `StatsEntry` struct . It looks like this two fields are the same :
`Container` : https://github.com/containerd/nerdctl/blob/master/pkg/statsutil/stats.go#L31
`ID` : https://github.com/containerd/nerdctl/blob/master/pkg/statsutil/stats.go#L29

we may keep only one identifier no need to have a duplicate identifier field

Reply:
> we may keep only one identifier no need to have a duplicate identifier field. This can be in other PR

Yes!



> We cannot have an empty ID.

Nope, after https://github.com/containerd/nerdctl/pull/1416/files#diff-8541fa08e70d092bb039071593918dac552c3cbf1088d15e71d0f6117ad4b249R471-R474, we may get empty ID here because of https://github.com/containerd/nerdctl/blob/master/cmd/nerdctl/stats.go#L408-L409